qatrack/faults/views.py

```
@@ -262,13 +264,41 @@ def get_context_data(self, *args, **kwargs):
+        review_errors = {f.prefix: f.errors.get_json_data() for f in context_data['review_forms'] if f.errors}
+        context_data['review_errors_json'] = json.dumps(review_errors)
+        
+        review_forms_data = []
+        for f in context_data['review_forms']:
+            review_forms_data.append({
+                'prefix': f.prefix,
+                'group_name': f.fields['group'].initial,
+                'required': f.fields['reviewed_by'].required,
+                'users': [{'id': u.id, 'name': u.get_full_name() or u.username} for u in f.fields['reviewed_by'].queryset],
+                'selected': f.data.get(f.prefix + '-reviewed_by') if f.is_bound else f.initial.get('reviewed_by')
+            })
```

Ruff (0.15.15)

[warning] Use a list comprehension to create a transformed list

(PERF401)